res/res_stasis.c

```diff
@@ -1829,7 +1841,7 @@ int stasis_app_register_all(const char *app_name, stasis_app_cb handler, void *d
 	return __stasis_app_register(app_name, handler, data, 1);
 }
 
-void stasis_app_unregister(const char *app_name)
+static void __stasis_app_unregister(const char *app_name, int all_events)
```

Comment:
I think int unsubscribe_event_sources reflects what it is for better

Reply:
I'm removing stasis_app_unregister_all().